src/.schema.sql

```diff
@@ -0,0 +1,15 @@
+CREATE SCHEMA kubercade AUTHORIZATION nrhwqomw
+
+  CREATE TABLE high_score_table (
+    game_index  integer,
+    name        varchar(80),
+    score       integer NOT NULL,
```

Comment:
I'm assuming for now all our scores will be integers. Not sure if this might create a problem in the future, but the games I looked at that we considered integrating all seemed to have integer scores.

Reply:
We can just only do games with integer scores - at worst we can just count the number of wins.